news/views.py

```diff
@@ -526,6 +526,15 @@ def newsletters(request):
     })
 
 
+@require_safe
+@cache_page(30)
+def healthz(request):
+    """for use with healthchecks. wrapped with `cache_page` to test redis cache."""
```

Comment:
This means that it will only actually check the DB every 30 minutes. It would make more sense to me if we hit both redis and the db within the uncached view.